<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BacktracePlugin::DialogBacktrace</class>
 <widget class="QDialog" name="BacktracePlugin::DialogBacktrace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Call Stack</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidgetCallStack">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Call From</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Return To</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BacktracePlugin::DialogBacktrace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>538</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BacktracePlugin::DialogBacktrace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
